app/src/main/java/com/github/k409/fitflow/ui/FitFlowApp.kt

```diff
@@ -98,6 +102,25 @@ fun FitFlowApp(
 
     val snackbarHostState = remember { SnackbarHostState() }
 
+    val context = LocalContext.current
+    // ImageLoader has its own memory cache, disk cache, and OkHttpClient so it should be initialized only once in app
+    ImageLoader.Builder(context)
```

Comment:
Move this to FitFlowApplication. In Application class, implement the ImageLoaderFactory interface and override the newImageLoader() function. In method should be this logic.